src/main/resources/config/tr.yml

```diff
@@ -0,0 +1,359 @@
+DiscordConsoleChannelBlockBots: true
+
+# Sohbet kanalı komutu yürütme komutu
+# Bu seçenekler, bir komutu çalıştırmak için "!c kick Notch" veya önek ne olursa olsun deme yeteneğini kontrol eder.
+# Konsol olarak kayıtlı bir sohbet kanalından.
+#
+# DiscordChatChannelConsoleCommandEnabled: Bir sohbet kanalından gelen konsol komutlarına izin verilip verilmeyeceği.
+# DiscordChatChannelConsoleCommandNotifyErrors: İzni olmayan bir komutu izinsiz çalıştırmaya çalışan bir kullanıcıya gönderilip gönderilmeyeceği
+# DiscordChatChannelConsoleCommandPrefix: Konsol komutları için kullanılacak önek. Örneğin. "!c tps"
+# DiscordChatChannelConsoleCommandRolesAllowed: Sohbet kanalından sunucu komutlarını yürütmesine izin verilen kullanıcı rolleri
+# DiscordChatChannelConsoleCommandWhitelist: Çalıştırılabilecek komutların listesi DiscordChatChannelConsoleCommandPrefix
+# DiscordChatChannelConsoleCommandWhitelistBypassRoles: Beyaz listeyi atlayan rollerin listesi
+# DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: Beyaz liste komutu bunun yerine kara liste görevi görmeli mi?
+# DiscordChatChannelConsoleCommandExpiration: Gönderilen bir komut çıktısının bot tarafından otomatik olarak kaldırılmasına kadar geçen saniye cinsinden süre. Süre sonunu devre dışı bırakmak için 0'a ayarlayın.
+# DiscordChatChannelConsoleCommandExpirationDeleteRequest: Komutu ilk veren kişinin mesajının silinip silinmeyeceği
+#
+DiscordChatChannelConsoleCommandEnabled: true
+DiscordChatChannelConsoleCommandNotifyErrors: true
+DiscordChatChannelConsoleCommandPrefix: "!c"
+DiscordChatChannelConsoleCommandRolesAllowed: ["Owner", "Developer"]
+DiscordChatChannelConsoleCommandWhitelist: ["say", "lag", "tps"]
+DiscordChatChannelConsoleCommandWhitelistBypassRoles: ["Owner", "Developer"]
+DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: false
+DiscordChatChannelConsoleCommandExpiration: 0
+DiscordChatChannelConsoleCommandExpirationDeleteRequest: true
+
+# Sohbet kanalı oynatıcı listesi komutu
+# Oyuncu listesi komutuna ilişkin tüm yapılandırma öğeleri
+#
+# DiscordChatChannelListCommandEnabled: Komutun etkin olup olmadığı
+# DiscordChatChannelListCommandMessage: İnsanların oyuncu listesini almak için yazabilecekleri komut
+# DiscordChatChannelListCommandExpiration: Gönderilen oyuncu listesi mesajının bot tarafından otomatik olarak kaldırılmasına kadar geçen saniye cinsinden süre. Süre sonunu devre dışı bırakmak için 0'a ayarlayın.
+# DiscordChatChannelListCommandExpirationDeleteRequest: Oyuncu listesini ilk talep eden kişinin mesajının silinip silinmeyeceği
+#
+DiscordChatChannelListCommandEnabled: true
+DiscordChatChannelListCommandMessage: "playerlist"
+DiscordChatChannelListCommandExpiration: 10
+DiscordChatChannelListCommandExpirationDeleteRequest: true
+
+# Sohbet kanalı kara listeye alınan ifadeler ve normal ifadeler
+#
+# DiscordChatChannelGameFilters: Discord'a gönderilen sohbet mesajlarına normal ifade filtreleri uygulanacak, sonuç boşsa mesaj hiç gönderilmeyecek
```

Comment:
Regex shouldn't be translated for users to search online easily as no one uses "normal ifadeler" to translate "regex"

Reply:
Even the auto translate doesn't translate it at line 258